src/HTTPFile.cc

```diff
@@ -101,9 +137,7 @@ int HTTPFile::Open(const char *path, int Oflag, mode_t Mode, XrdOucEnv &env) {
 		configured_hostname = m_oss->getStoragePrefix();
 	}
 
-	//
```

Comment:
Please rebase out spurious whitespace changes -- or, if you want to keep it, do it a sa separate whitespace commit.